### PR TITLE
Add more checks

### PR DESCRIPTION
**Pull request by slesaad**
Addresses: #55, #56, #68 and #69

### CODE REVIEW

File: pyQuARC/code/constants.py [hunk 1]
```diff
@@ -59,5 +59,8 @@
     "instruments": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/instruments?format=csv",
     "projects": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/projects?format=csv",
     "granuledataformat": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/granuledataformat?format=csv",
-    "platforms": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/platforms?format=csv"
+    "platforms": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/platforms?format=csv",
```
Comment (code-geek) [inline pyQuARC/code/constants.py:62]: Looks like we could put the base URL in a constant.

File: pyQuARC/code/constants.py [hunk 2]
```diff
@@ -59,5 +59,8 @@
...
+    "rucontenttype": "https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/rucontenttype?format=csv"
 }
+
+CRM_URL = "https://cmr.earthdata.nasa.gov"
```
Comment (code-geek) [inline pyQuARC/code/constants.py:66]: CRM -> CMR

File: pyQuARC/code/datetime_validator.py [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import pytz
 import re
+import requests
 
 from datetime import datetime
 
 from .base_validator import BaseValidator
+from .constants import CRM_URL
```
Comment (code-geek) [inline pyQuARC/code/datetime_validator.py:8]: CRM-> CMR

File: pyQuARC/code/datetime_validator.py [hunk 4]
```diff
@@ -73,3 +75,47 @@ def compare(first, second, relation):
...
+        Args:
+            ending_datetime (str): EndingDatetime string
+            collection_shortname (str): ShortName of the parent collection
+
+        Returns:
+            (dict) An object with the validity of the check and the instance
+        """
+        granules = requests.get(f'{CRM_URL}/search/granules.json?short_name={collection_shortname}&sort_key[]=-end_date').json()
+        last_granule = granules['feed']['entry'][0]
+        last_granule_endtime = last_granule['time_end']
```
Comment (xhagrg) [inline pyQuARC/code/datetime_validator.py:94]: Should we not handle this gracefully? If no results were returned?

File: pyQuARC/code/datetime_validator.py [hunk 5]
```diff
@@ -73,3 +75,47 @@ def compare(first, second, relation):
...
+        
+        return {
+            "valid": ending_datetime == last_granule_endtime,
+            "value": (ending_datetime, last_granule_endtime)
+        }
+
+    @staticmethod
+    @if_arg
+    def validate_beginning_datetime_against_granules(beginning_datetime, collection_shortname):
```
Comment (xhagrg) [inline pyQuARC/code/datetime_validator.py:103]: The previous method and this method are awfully similar. Can we not handle it someway else?

**Comment by slesaad** [comment]
Thanks for making the changes @xhagrg, they look good

**Pull request closed without merging**